src/main/Token.js

```diff
@@ -125,7 +126,9 @@ class Token {
                     engine,
                     this._developerKey,
                     this._globalRpcErrorCallback);
-            await member.addAlias(alias);
+            if (Object.keys(alias).length !== 0) {
```

Comment:
My JS skills are so weak that I'm not sure what you have in mind by an empty object. {}? false? anyhow... Probably a good idea to add a no-alias createMember invocation to tests/library.spec.js or sdk-js-tests/something to show me.

Reply:
condition here is testing for {}, and yeah I can add something to the tests.

Reply:
Yes a test, or check for null / undefined would be good too, for the most flexibility.